docs/running_a_service.md

```diff
@@ -0,0 +1,246 @@
+- with a local, in-memory signer service rather than using Notary signer,
+- using a local, in-memory TUF metadata store rather than using MySQL
+- produce JSON-formatted logs
+
+One way to do this would be:
+
+1. Generate your own TLS certificate and key as `server.crt` and `server.key`,
+	and put them in the directory `/tmp/server-configdir`.
+
+2. Write the following configuration file to `/tmp/server-configdir/config.json`:
+
+		{
+		  "server": {
+		    "http_addr": ":4443",
+		    "tls_key_file": "./server.key",
+			"tls_cert_file": "./server.crt"
+		  },
+		  "trust_service": {
+		    "type": "remote",
+		    "hostname": "notarysigner",
+		    "port": "7899",
+		    "tls_ca_file": "./root-ca.crt",
+		    "key_algorithm": "ecdsa",
+		    "tls_client_cert": "./notary-server.crt",
+		    "tls_client_key": "./notary-server.key"
+		  },
+		  "storage": {
+		    "backend": "mysql",
+		    "db_url": "server@tcp(mysql:3306)/notaryserver?parseTime=True"
+		  }
+		}
+
+	Note that we are including a remote trust service and a database storage
+	type in order to demonstrate how environment variables can override
+	configuration parameters.
+
+3. Run the following command (assuming you've already built or pulled a Notary server docker image):
+
+		$ docker run \
+			-p "4443:4443" \
+			-v /tmp/server-configdir:/etc/docker/notary-server/ \
+			-e NOTARY_SERVER_TRUST_SERVICE_TYPE=local \
+			-e NOTARY_SERVER_STORAGE_BACKEND=memory \
+			-e NOTARY_SERVER_LOGGING_LEVEL=debug \
+			notary_server \
+				-config=/etc/docker/notary-server/config.json \
+				-logf=json
+		{"level":"info","msg":"Version: 0.2, Git commit: 619f8cf","time":"2016-02-25T00:53:59Z"}
+		{"level":"info","msg":"Using local signing service, which requires ED25519. Ignoring all other trust_service parameters, including keyAlgorithm","time":"2016-02-25T00:53:59Z"}
+		{"level":"info","msg":"Using memory backend","time":"2016-02-25T00:53:59Z"}
+		{"level":"info","msg":"Starting Server","time":"2016-02-25T00:53:59Z"}
+		{"level":"info","msg":"Enabling TLS","time":"2016-02-25T00:53:59Z"}
+		{"level":"info","msg":"Starting on :4443","time":"2016-02-25T00:53:59Z"}
+
+You can do the same using [Docker
+Compose](https://docs.docker.com/compose/overview/) by setting volumes,
+environment variables, and overriding the default command for the Notary server
+containers in the Compose file.
+
+
+## Recommendations for deploying in production
+
+When moving from development to production there are a number of considerations
+that must be made to ensure security and scalability.
+
+### Certificates
+
+The Notary repository includes sample certificates in the fixtures directory.
+When you initialize a development service using the provided docker-compose.yml
+file, these sample certificates are used to create a more production like
+environment.
+
+YOU MUST ACQUIRE YOUR OWN CERTIFICATES TO USE IN A PRODUCTION DEPLOYMENT.
+
+The private key files in the Notary repository are obviously public knowledge
```

Comment:
nit: could we change this to "sample private key files" to make it more explicit?